modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -125,12 +119,12 @@ std::vector<std::string> deserialize(const std::vector<char> &bytes) {
 /**
  * @brief Deserialize GCompileArgs which types were specified in the template from a byte array.
  *
- * @note S11N template specialization must be provided to make a custom type
+ * @note cv::gapi::s11n::detail::S11N template specialization must be provided to make a custom type
  * in GCompileArgs deserializable.
  *
  * @param bytes vector of bytes to deserialize GCompileArgs object from.
  * @return GCompileArgs object.
- * @see GCompileArgs S11N
+ * @see GCompileArgs cv::gapi::s11n::detail::S11N
```

Comment:
hmm it is strange to see `detail::` thing in the documentation -- is our overloadable `S11N` structure still there?

Reply:
Yes, it's still there